docs/src/routes/docs/components/typography.svx

```diff
@@ -0,0 +1,89 @@
+---
+name: Typography
+---
+
+<script>
+  import { H1, H2, H3, Headline, Subhead, Label } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+</script>
+
+# Typography
+
+Typography elements with embedded responsive styles.
+
+Also comes with a `<TypographyStyles>` component, which only adds global styles, setting the `$font` for the whole page, coloring links, etc. A good place to put it for Sapper, for example, would be anywhere in the main [layout file](https://sapper.svelte.dev/docs#Layouts).
```

Comment:
I think an example of this `<TypographyStyles>` component has to be shown. I cannot understand its purpose or how to use it just from this page

Reply:
Sigh. Yeah, this is a major weakness. For some reason I actually feel a lot of dissatisfaction with those docs in general, but I do not yet know how to improve. In this case as well, I don't know how to improve this. Honestly, the whole idea of the `TypographyStyles` component kinda blows, and I believe we didn't even use it in Innopoints, so maybe it could just be removed altogether

Reply:
by _"those docs in general"_, are you referring to the typography docs or all components' docs? If all of them, why so?
About the `<TypographyStyles>` component (of whose existence I was previously unaware), I see that all it does is include some styles. They are not even that many styles and I think we can let go of it entirely. However, if you believe it might be useful, we can export the SCSS file itself directly such that users can include it in their project as they want (in their own SCSS or in the preprocessor's `includePaths`) as it is not really a component, but simply a bunch of styles made `:global` (imo, the purpose of a component should not be for side-effects purely, if at all)

Reply:
All of them. [Here's a framework](https://ant.design/components/button/), for example, that has lots of examples in a rather easy to consume format. Originally I had a lot of faith in the showcase section, but it has proven to be unsuitable in the general case, and not even useful. Examples in that framework, for instance, show actual real-world usage of the components, and the showcase is a very artificial, dead way of displaying them, aiming to showcase as much stuff as possible in as little space, which harms it even more. The prop tables and generally the structure of the documentation pages also feels dead to me, like we're taking consistency over clarity, which makes the docs relentlessly consistent but hard to read and learn from, makes them more machine than human.

As for TypographyStyles, yes, fine, let's just remove this component.

Reply:
Don't beat yourself too hard over this. I think they look great and easy to understand, and our components are simple enough that we do not need that much examples. I think we should just go with it and wait for any feedback